decisions/imports-ts-suppressions.md

```diff
@@ -0,0 +1,80 @@
+# Untyped import statements TypeScript suppressions
+
+## TL;DR
+
+When using TypeScript in a file that imports from any untyped add a suppression like the below:
+
+```ts
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { cssModules } from 'bpk-react-utils';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { helper } from 'some-external-untyped-package';
+```
```

Comment:
Shall we add stubs in the type definitions like Banana instead?

Reply:
See the full doc - we cannot as these stubs are not picked up by consumers of Backpack, stopping `npm run start` / `build` working.

Reply:
Continuing in https://github.com/Skyscanner/backpack/pull/2535#discussion_r952585158